src/main/jhipster/src/main/webapp/app/germplasm-details/inventory/inventory-pane.component.ts

```diff
@@ -28,6 +30,9 @@ export class InventoryPaneComponent implements OnInit {
     }
 
     ngOnInit(): void {
+        // Call this service but do nothing. This is just to make sure that if token is invalid (via AuthExpiredInterceptor), the page will be redirected to login.
+        this.germplasmService.getGermplasmById(this.germplasmDetailsContext.gid).toPromise().then((value) => {
```

Comment:
As discussed, call endpoint /bmsapi/validateToken instead here (better create a service for it)